### PR TITLE
Support for Python 2.7 and 3.4. Fixes #35

### LINKED ISSUE #35: Python Compatibility
pycodestyle provides support for 2.7, 3.4 and PyPy. Was this intentionally left off for this package? It would be beneficial to provide the support so other packages can move off the legacy pytest-pep8 package and use supported pycodestyle.

### PR DESCRIPTION
**Pull request by fschulze**
Fixes #35

### CODE REVIEW

**Comment by smarlowucf** [comment]
:+1:

**Comment by henry0312** [comment]
Thank you for your contribution, but I'm sorry that I will never support 2.7 and 3.4, because

1. I don't use those versions and I can't support them.
1. I want Python 2 users to upgrade thier Python to the latest version of 3.
1. Python 3.4 will reach its end of life soon and pytest-codestyle supports 3.5.

cf. https://github.com/henry0312/pytest-codestyle/pull/5
cf. [PEP 429 -- Python 3.4 Release Schedule](https://www.python.org/dev/peps/pep-0429/)
cf. [Plan for dropping Python 2.7 — pandas 0.23.4 documentation](https://pandas.pydata.org/pandas-docs/stable/install.html#plan-for-dropping-python-2-7)

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,12 @@
 language: python
 python:
+  - "2.7"
+  - "3.4"
   - "3.5"
   - "3.6"
+  - "nightly"
```
Comment (henry0312) [inline .travis.yml]: good!

File: setup.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 from setuptools import setup
 
 version = open('VERSION').read().strip()
-long_description = open('README.md', encoding='utf-8').read()
+long_description = open('README.md', 'rb').read().decode('utf-8')
```
Comment (henry0312) [inline setup.py]: 👍

**Comment by smarlowucf** [comment]
> but I'm sorry that I will never support 2.7 and 3.4

This is unfortunate. I understand the reasoning but it forces any package that requires `pytest-codestyle` to also drop support for 2.7 and 3.4.

Personally I will switch to pytest-flake8 which has the needed support. I agree it's nice to move everything to latest Python3.x but it's also nice to support all active Python versions.

**Pull request closed without merging**